<launch>
    <param name="use_sim_time" value="true"/>
    <rosparam  command="load" file="$(find roborts_slam)/config/simulatin_param.yaml" />

    <node name="roborts_slam_node" pkg="roborts_slam"  type="roborts_slam_node" output="screen" >
        <remap from="scan" to="scan" />
    </node>

    <node pkg="stage_ros" type="stageros" name="stageros" args="$(find roborts_slam)/worlds/icra.world" respawn="false" >
        <remap from="base_scan" to="scan"/>
        <param name="base_watchdog_timeout" value="0.5" />
    </node>

    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find roborts_slam)/rviz/rviz.rviz" />

</launch>
